### ISSUE TITLE
[BUG] Not Working with versions 5.0.0 and 4.0.0 but runs ok with 3.1.0 of CommunityToolKit.Maui.Core

### ISSUE THREAD

**Issue by CriptyX** [CommunityToolkit/Maui#1057]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When the application starts in Android, an exception is thrown since version 4.0.0, however with version 3.1.0 everything works correctly.
The Error is:
System.MissingFieldException: 'Field not found: int .Animation.linear_indeterminate_line1_head_interpolator Due to: Could not find field in class'

### Expected Behavior

Have the application start normally.

### Steps To Reproduce

Application only for iOS and Android. in .Net7

Application with the following Nuget packages:

CommunityToolkit.Maui (3.0.0)
CommunityToolkit.Maui.Core (3.0.0)
CommunityToolkit.Maui.Markup (3.0.0)
Microsoft.Extensions.Configuration (7.0.0)
Microsoft.Extensions.DependencyInjection (7.0.0)
Microsoft.Extensions.Localization (7.0.3) 
Microsoft.Extensions.Loggin( 7.0.0)
Microsoft.Extensions.Loggin.Abstractions (7.0.0)
Microsoft.Extensions.Loggin.Debug (7.0.0)
Microsoft.Maui.Controls.Maps(7.0.59)
Microsoft.Maui.Graphics (7.0.59)
Newtonsoft.Json (13.0.2)
Onion.Maui.GoogleMaos(5.0.0)
Xamarin.Android.glide (4.13.2.2)
Xamarin.AndroidX.Browser(1.4.0.3)
Xamarin.AndroidX.Legacy.Support.V4(1.0.0.15)
Xamarin.AndroidX.Lifecycle.LiveData(2.5.1.1)
Xamarin.AndroidX.Navigation.Common(2.5.2.1)
Xamarin.AndroidX.Navigation.Fragment(2.5.2.1)
Xamarin.AndroidX.Navigation.Runtime(2.5.2.1)
Xamarin.AndroidX.Navigation.UI(2.5.2.1)
Xamarin.AndroidX.Security.SecurityCrypto(1.1.0-alpha03)
Xamarin.Google.Android.Material(1.7.0)
Xamarin.Google.Crypto.Tink.Android(1.7.0.1)

Start application, application fails.

### Link to public reproduction project repository

https://github.com/CriptyX/TestCommunityToolKit

### Environment

```markdown
- .NET MAUI CommunityToolkit: From 3.0.0
- .NET MAUI CommunityToolkit.Core: From 4.0.0
- OS: Android
- .NET MAUI: 7.0.59
```

### Anything else?

WorkArround: stay in version 3.1.0

**Comment by CriptyX** [comment]
Thanks for your prompt reply.

This repository is just a demo to prove that it fails. In the "real" application I need the ComunityToolKit.Maui, and it has as requirement the ComunityToolKit.Maui.Core, to find that it fails I have been adding, removing, and the failure gives me for the ComunityToolKit.Maui.Core.

I must add, that I have not tested against the emulator, always against a real device (a Samsung Galaxy A52s 5g with Android 13 and One UI 5.0 version and al updates). 

After seeing that you have tested it on an emulator, I have tested it too to check, both the test program and the "real" one and both have worked. I have taken another test terminal (A Realme GT also with Android 13 and RealMe UI 4.0) and it also worked. I took another one, a Mi11, and it also worked correctly.

I went back to the Samsung Galaxy and it failed. So the problem has to be with the version from 3.1.0 to up on the Samsung Galaxy A52s 5G device. The funny thing is that when I put the 3.0.0 version of the ComunityToolKit.Maui.Core it doesn't give any problem.

I know that if it only fails in a specific model of phones may not be a priority, but Samsung is a fairly common brand, and I've already had bad experiences, precisely with Samsung to deploy an app that works well on all devices except a few Samsung models...

I hate Samsung almost as much as I hate Apple....

Thanks in Advance.

**Comment by VladislavAntonyuk** [comment]
the only thing I can recommend is to make a clean build and reinstall the app (delete, then install). I checked on Samsung Galaxy Tab S and also don't have an exception. You shouldn't explicitly add ComunityToolKit.Maui.Core in your project. it is automatically resolved by ComunityToolKit.Maui

**Comment by CriptyX** [comment]
I have tried, but nothing. I guess I'll stick with CommunityToolKit 3.0.0 and try again when another version comes out. In any case, the real application is not planned for a couple of months, another version may come out and work on all mobiles...

I've tried with a couple of other mobiles and it works... it's only with the Samsung that it fails (And I've given it a factory reset to try... Although the mobile was new and I had just started using it in this development...) Anyway, it will be one of those computer mysteries.

Thank you so much.
